open_spiel/python/algorithms/nash_averaging.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Nash averaging.
+
+  Based on https://arxiv.org/pdf/1806.02643.pdf
+  An axiomatic strategy evaluation metric for 
+  Agent-vs-Agent or Agent-vs-Task two-player zero-sum games
+"""
+
+from cvxopt import solvers, matrix, spdiag, spmatrix, log
+import numpy as np
+
+from open_spiel.python.egt.utils import game_payoffs_array
+
+
+def max_entropy_symmetric_nash(p_mat, eps=0.0):
```

Comment:
Is this intended to be a public function?

If not, prepend with an underscore: _max_entropy_symmetric_nash